configure.ac

```diff
@@ -563,7 +563,7 @@ if test "$with_libpam" != "no"; then
 	fi
 
 
-	if test "X$pam_lib$pam_headers_found" = "Xyesyes" -a "$pam_conv_function" != "no" ; then
+	if test "X$pam_lib$pam_headers_found" = "Xyesyes" && test "$pam_conv_function" != "no" ; then
```

Comment:
Thanks!

For this patch (1db760b4057 ("configure: fixed outdated and non-portable 'test' syntax ")):

```
Reviewed-by: Alejandro Colomar <alx@kernel.org>
```